src/BenchmarkDotNet.Core/Diagnosers/CompositeDiagnoser.cs

```diff
@@ -19,13 +19,13 @@ public CompositeDiagnoser(params IDiagnoser[] diagnosers)
 
         public IColumnProvider GetColumnProvider() 
             => new CompositeColumnProvider(diagnosers.Select(d => d.GetColumnProvider()).ToArray());
-
+#if !UAP
```

Comment:
What if Diagnoser's interface exploit something like IHost interface, that would be implemented in OS-specific terms.